samples/dotnetsay/README.md

```diff
@@ -58,10 +58,10 @@ The `PackAsTool` property in the project file makes it a global tool, as you can
 You can install the tool using the following command.
 
 ```console
-dotnet install tool -g dotnetsay
+dotnet tool install -g dotnetsay
 ```
 
-For Preview 1, defining the source during installation doesn't work correctly, so you need a nuget.config file to test your new tool without deploying it to a NuGet feed. You can do this by placing this nuget.config in your project directory that looks similar to the following example:
+For Preview 2, defining the source during installation doesn't work correctly, so you need a nuget.config file to test your new tool without deploying it to a NuGet feed. You can do this by placing this nuget.config in your project directory that looks similar to the following example:
```

Comment:
I believe this has been fixed. @wli3 @peterhuene to comment.

Reply:
This should now work in preview 2.